lib/active_fedora/schema_application.rb

```diff
@@ -0,0 +1,16 @@
+module ActiveFedora
+  ##
+  # Implement the .apply_schema method from ActiveTriples to allow for
+  # externally defined schemas to be put on an AF::Base object.
+  module SchemaApplication
+    extend ActiveSupport::Concern
+
+    module ClassMethods
+      def apply_schema(schema, strategy=ActiveFedora::SolrApplicationStrategy.new([:symbol]))
```

Comment:
I'm assuming the common use cases for a schema are searching, faceting, and displaying. :symbol solves the displaying/faceting/exact search cases. I'd be happy to do :symbol and :stored_searchable.